libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Memory/Expressions/ArrayExpression.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections;
+using System.Collections.Generic;
+using Microsoft.Bot.Builder.Dialogs.Adaptive.Converters;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Bot.Builder.Dialogs.Adaptive
+{
+    /// <summary>
+    /// ArrayExpression - represents a property which is either a value of array of T or a string expression to bind to a array of T.
+    /// </summary>
+    /// <typeparam name="T">type of object in the array.</typeparam>
+    /// <remarks>String values are always be interpreted as an expression, whether it has '=' prefix or not.</remarks>
```

Comment:
>be [](start = 42, length = 2)

Remove.

Reply:
Looks like this is in all of the expressions.

---
In reply to: [367275514](https://github.com/microsoft/botbuilder-dotnet/pull/3253#discussion_r367275514) [](ancestors = 367275514)